lib/cf/applications/src/index.js

```diff
@@ -53,12 +53,13 @@ const db = throttle(retry(breaker(batch(
     'abacus-cf-bridge'))
 ))));
 
-const minIntervalTime = parseInt(process.env.MIN_INTERVAL_TIME) || 5000;
+const minIntervalTime = parseInt(process.env.MIN_INTERVAL_TIME) || 30000;
 const maxIntervalTime = parseInt(process.env.MAX_INTERVAL_TIME) || 240000;
 const guidMinAge = parseInt(process.env.GUID_MIN_AGE) || 60000;
 const orgsToReport = process.env.ORGS_TO_REPORT ?
   JSON.parse(process.env.ORGS_TO_REPORT) : undefined;
 const lastRecordedGuid = process.env.LAST_RECORDED_GUID;
+const resultsPerPage = parseInt(process.env.PAGE_SIZE) || 100;
```

Comment:
We already have `PAGE_SIZE` for DB paging and thus cannot change both independently.
What about prefixing with `API_`?